pilot/pkg/networking/plugin/authn/authentication.go

```diff
@@ -155,14 +162,56 @@ func ConvertPolicyToJwtConfig(policy *authn.Policy) *jwtfilter.JwtAuthentication
 	return ret
 }
 
+// ConvertPolicyToAuthNFilterConfig returns authn filter config corresponding for the input policy.
```

Comment:
returns _an_ authentication filter config corresponding _to_ the input policy

Reply:
Done